File: CS/Q147080/Tab/Styles/MyOffice2003ViewInfoRegistrator.cs
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.ViewInfo;
using DxSample.Tab.ViewInfo;

namespace DxSample.Tab.Styles {
    public class MyOffice2003ViewInfoRegistrator : Office2003ViewInfoRegistrator {
        public MyOffice2003ViewInfoRegistrator() : base() { }
        public override BaseTabHeaderViewInfo CreateHeaderViewInfo(BaseTabControlViewInfo viewInfo) {
            return new MyOffice2003TabHeaderViewInfo(viewInfo);
        }
    }
}

File: VB/Q147080/Tab/Styles/MyOffice2003ViewInfoRegistrator.vb
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.ViewInfo
Imports DxSample.Tab.ViewInfo

Namespace DxSample.Tab.Styles
    Public Class MyOffice2003ViewInfoRegistrator
        Inherits Office2003ViewInfoRegistrator

        Public Sub New()
            MyBase.New()
        End Sub
        Public Overrides Function CreateHeaderViewInfo(ByVal viewInfo As BaseTabControlViewInfo) As BaseTabHeaderViewInfo
            Return New MyOffice2003TabHeaderViewInfo(viewInfo)
        End Function
    End Class
End Namespace
